src/Entity/Attributes/SharedFolder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Entity\Attributes;
+
+use App\Entity\Dossier;
+use App\Repository\SharedFolderRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: SharedFolderRepository::class)]
+#[ORM\Table(name: 'shared_folder')]
+class SharedFolder extends SharedPersonalData
+{
+    #[ORM\ManyToOne(inversedBy: 'sharedFolders')]
+    #[ORM\JoinColumn(nullable: false)]
+    private ?Dossier $folder = null;
```

Comment:
I think we should have the same attributes in `SharedDocument` and `SharedFolder`